css/styles.less

```
@@ -0,0 +1,854 @@
+@import "global.less";
+
+// #region header
+header {
+  position: sticky;
+  position: -webkit-sticky; /* Safari */
+  top: 0;
+  width: 100%;
+  height: @nav-height;
+  background-color: white;
+  box-shadow: 0 4px 6px -8px black;
+  z-index: 1000;
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    height: 100%;
+    padding-top: .45em;

```

making css suggestions for the header

```suggestion
display: flex;
padding-top: .45em;
flex-wrap: wrap;
```